.github/pull_request_template.md

```diff
@@ -13,18 +11,55 @@
 
 ## Issue type checklist
 
-<!--- What type of change are you submitting? Check the boxes that apply: -->
+<!--- What type of change are you submitting? Put an x in boxes that apply: -->
```

Comment:
This still eludes to multiples of these being an option. That is not possible from a semver standpoint, and our policy is one PR per issue.

Reply:
Changing to :  "What type of change are you submitting? Put an x in the box that applies:"